docker-compose.local.yaml

```diff
@@ -60,9 +60,11 @@ services:
 
   adminer:
```

Comment:
i would prefer if this service def inherits from `docker-compose.dev.yaml` to avoid redefining all the fields. I think other things here do the same?

Reply:
Good point thanks